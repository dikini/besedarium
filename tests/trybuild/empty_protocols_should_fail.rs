use besedarium::*;

// All trybuild compile-fail tests in this file have been temporarily disabled to stabilize the test base for the TInteract refactor.

// Should fail: tchoice! and tpar! with no branches
// These should not compile, as empty protocols are not allowed.
// Uncomment one at a time to see the error.
// type EmptyChoice = tchoice!(Http;);
// type EmptyPar = tpar!(Http;);
