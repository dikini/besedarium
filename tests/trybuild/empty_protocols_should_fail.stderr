warning: unused import: `playground::*`
 --> tests/trybuild/empty_protocols_should_fail.rs:1:5
  |
1 | use playground::*;
  |     ^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/trybuild/empty_protocols_should_fail.rs:1:19
  |
1 | use playground::*;
  |                   ^ consider adding a `main` function to `$DIR/tests/trybuild/empty_protocols_should_fail.rs`
